system/conftest.py

```diff
@@ -199,6 +199,11 @@ def wrapped(nodes_num, request):
         if (request.node.nodeid in _failed_nodes) and request.config.getoption("gatherlogs"):
             logs_dir = os.path.join(request.config.getoption("logsdir"), session_name, request.node.nodeid)
 
+        # fix for prod case
+        if request.node.name == 'test_case_complex_pool_operations':
```

Comment:
i think we need to move `test_case_complex_pool_operations` test into separate module and redefine _docker_teardown fixture for using this test's specific behavior.